mods/ra/rules/aircraft.yaml

```diff
@@ -340,43 +340,46 @@ HIND:
 	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
 	Buildable:
 		Queue: Aircraft
-		BuildAtProductionType: Helicopter
-		BuildPaletteOrder: 20
-		Prerequisites: ~hpad, ~techlevel.medium
-		Description: Helicopter gunship armed\nwith dual chainguns.\n  Strong vs Infantry, Light armor\n  Weak vs Tanks, Aircraft
+		BuildAtProductionType: Plane
+        BuildPaletteOrder: 35
```

Comment:
There is a tabbing error here, also please use 40 and move everything else up instead of using 35.

Reply:
To be fair: IIRC the original motivation of having the build order as multiples of 10 was exactly to allow inserting other actors in-between without having to adjust all following actors.

Reply:
I think the values in main mods should stay as multipiles of 10, but it helps with the custom maps, which i think should be the reasoning for it.

Reply:
Moved Longbow and Mig up by 10 to fit Hind in 40